juniper_official/Protocols/Isis/isis-statistics.rule

```diff
@@ -0,0 +1,378 @@
+/*
+ * Detects ISIS packet drops and notifies when anomalies are found.
+ * Two input control detection
```

Comment:
Can we extend this to include the chip level RCA rules that Dan developed for OSPF - but changed for ISIS?

Reply:
It can be done post 2.0.0. Please approve current rules.